### PR TITLE
Rollout improved example converter

### PR DESCRIPTION
**Pull request by t0yv0**
Rolls out improved TF example converter.

### CODE REVIEW

**Comment by VenelinMartinov** [comment]
This one has quite a few new examples but has also lost some.

File: sdk/go/github/actionsOrganizationSecret.go [hunk 1]
```diff
@@ -14,90 +14,6 @@ import (
...
-//				Visibility:     pulumi.String("private"),
-//				PlaintextValue: pulumi.Any(_var.Some_secret_string),
-//			})
-//			if err != nil {
-//				return err
-//			}
-//			_, err = github.NewActionsOrganizationSecret(ctx, "exampleSecretIndex/actionsOrganizationSecretActionsOrganizationSecret", &github.ActionsOrganizationSecretArgs{
-//				SecretName:     pulumi.String("example_secret_name"),
-//				Visibility:     pulumi.String("private"),
-//				EncryptedValue: pulumi.Any(_var.Some_encrypted_secret_string),
```
Comment (t0yv0) [inline sdk/go/github/actionsOrganizationSecret.go:41]: Suspect it fell out because of undefined _var.Some_encrypted_secret_string - checking

Comment (t0yv0) [reply sdk/go/github/actionsOrganizationSecret.go:41]: Hmm

```
{
  "ExampleName": "#/resources/github:index/actionsOrganizationSecret:ActionsOrganizationSecret",
  "OriginalHCL": "resource \"github_actions_organization_secret\" \"example_secret\" {\n  secret_name     = \"example_secret_name\"\n  visibility      = \"private\"\n  plaintext_value = var.some_secret_string\n}\n\nresource \"github_actions_organization_secret\" \"example_secret\" {\n  secret_name     = \"example_secret_name\"\n  visibility      = \"private\"\n  encrypted_value = var.some_encrypted_secret_string\n}\n",
  "ConversionResults": {
    "csharp": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "go": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "java": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "python": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "typescript": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "yaml": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    }
  }
}

```

Comment (t0yv0) [reply sdk/go/github/actionsOrganizationSecret.go:41]: https://github.com/pulumi/pulumi-converter-terraform/issues/146

**Comment by t0yv0** [comment]
```
jq '.|select(.ConversionResults.java.FailureSeverity==2)|.ConversionResults.typescript.FailureInfo' cov/byExample.json  | sort | uniq                                                                                                  ~/bugs/ci-mgmt-889/pulumi-github
""
"Duplicate resource \"github_actions_environment_secret\" configuration: A github_actions_environment_secret resource named \"example_secret\" was already declared at /e0.tf:1,1-62. Resource names must be unique per type in each module."
"Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module."
"Duplicate resource \"github_actions_secret\" configuration: A github_actions_secret resource named \"example_secret\" was already declared at /e116.tf:5,1-50. Resource names must be unique per type in each module."
"Duplicate resource \"github_codespaces_organization_secret\" configuration: A github_codespaces_organization_secret resource named \"example_secret\" was already declared at /e82.tf:1,1-66. Resource names must be unique per type in each module."
"Duplicate resource \"github_codespaces_secret\" configuration: A github_codespaces_secret resource named \"example_secret\" was already declared at /e58.tf:5,1-53. Resource names must be unique per type in each module."
"Duplicate resource \"github_codespaces_user_secret\" configuration: A github_codespaces_user_secret resource named \"example_secret\" was already declared at /e131.tf:5,1-58. Resource names must be unique per type in each module."
"Duplicate resource \"github_dependabot_organization_secret\" configuration: A github_dependabot_organization_secret resource named \"example_secret\" was already declared at /e53.tf:1,1-66. Resource names must be unique per type in each module."
"Duplicate resource \"github_dependabot_secret\" configuration: A github_dependabot_secret resource named \"example_secret\" was already declared at /e130.tf:5,1-53. Resource names must be unique per type in each module."
"Invalid multi-line string: Quoted strings may not be split over multiple lines. To produce a multi-line string, either use the \\n escape to represent a newline character or use the \"heredoc\" multi-line template syntax.; Missing newline after argument: An argument definition must end with a newline."
"Invalid multi-line string: Quoted strings may not be split over multiple lines. To produce a multi-line string, either use the \\n escape to represent a newline character or use the \"heredoc\" multi-line template syntax.; Unterminated template string: No closing marker was found for the string."
"pcl.BindProgram failed: example.pp:0,14-35: unknown function 'github:index:labels'; unknown function 'github:index:labels'"
```

**Pull request merged**